java/memory/memory-core/src/main/java/org/apache/arrow/memory/AllocationReservation.java

```diff
@@ -59,7 +84,8 @@ public interface AllocationReservation extends AutoCloseable {
   ArrowBuf allocateBuffer();
 
   /**
-   * Get the current size of the reservation (the sum of all the add()s).
+   * Get the current size of the reservation (the sum of all the add()s). Int version is deprecated,
+   * use getLongSize instead.
```

Comment:
I don't see getLongSize?

(Also for autocomplete we might prefer getSizeLong?)

Reply:
Oh sorry I removed it and forgot to update